<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="main_layout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="main_top_layout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="start_auto_scroll_btn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>开启自动滚轮</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="auto_send_btn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>自动发送</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="record_btn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>录制脚本</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="main_container_layout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="main_container_widget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="default_page">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>190</x>
                 <y>140</y>
                 <width>55</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>欢迎使用！</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="auto_send_form_widget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QFormLayout" name="auto_send_form_layout">
              <property name="horizontalSpacing">
               <number>20</number>
              </property>
              <property name="verticalSpacing">
               <number>20</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="auto_send_pos_label">
                <property name="text">
                 <string>坐标（x,y）</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="auto_send_pos_input">
                <property name="placeholderText">
                 <string>输入x,y坐标 或 ctrl+G获取坐标</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="auto_send_time_label">
                <property name="text">
                 <string>发送间隔（ms）</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="auto_send_time_input">
                <property name="text">
                 <string>1000</string>
                </property>
                <property name="placeholderText">
                 <string>请输入发送间隔</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="auto_send_content_label">
                <property name="text">
                 <string>发送内容</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="auto_send_content_input">
                <property name="placeholderText">
                 <string>请输入发送内容</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="auto_send_count_label">
                <property name="text">
                 <string>发送次数</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="auto_send_count_input">
                <property name="text">
                 <string>2</string>
                </property>
                <property name="placeholderText">
                 <string>请输入发送次数</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QPushButton" name="auto_send_get_post_btn">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>获取坐标（Ctrl+G）</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+G</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QPushButton" name="auto_send_submit_btn">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>发送（Ctrl+S）</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+S</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="record_form_widget">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QFormLayout" name="record_form_layout">
              <property name="horizontalSpacing">
               <number>20</number>
              </property>
              <property name="verticalSpacing">
               <number>20</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="record_type_label">
                <property name="text">
                 <string>录制类型</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="record_type_layout" stretch="0,1">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="record_type_checkbox_1">
                  <property name="text">
                   <string>鼠标</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">record_type_checkbox_group</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="record_type_checkbox_2">
                  <property name="text">
                   <string>键盘</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">record_type_checkbox_group</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="record_play_rate_label">
                <property name="text">
                 <string>播放倍率</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="record_play_rate_input">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="record_submit_btn">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>开始录制(CTRL+S)</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+S</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="record_play_btn">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>播放</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>start_auto_scroll_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickStartAutoScrollBtn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_send_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickAutoSendBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_send_get_post_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickAutoSendGetPosBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>-14</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_send_submit_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickAutoSendSubmitBtn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>record_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickRecordBtn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>record_submit_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickChangeRecord(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>-25</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>record_play_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClickPlayRecord(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleClickStartAutoScrollBtn(bool)</slot>
  <slot>handleClickAutoSendBtn()</slot>
  <slot>handleClickAutoSendGetPosBtn()</slot>
  <slot>handleClickAutoSendSubmitBtn(bool)</slot>
  <slot>handleClickRecordBtn(bool)</slot>
  <slot>handleClickChangeRecord(bool)</slot>
  <slot>handleClickPlayRecord(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="record_type_checkbox_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
